service/controller/controller.go

```diff
@@ -0,0 +1,133 @@
+package controller
```

Comment:
`prometheus-config-controller` doesn't create a TPR, so we decided to call it a `controller`. So, even though we use the 'operator' framework, i'm calling this package `controller`, (as opposed to `operator`, which feels plain wrong)

Reply:
This should be clearly stated in the `README` as it's an important explanation for the naming.

Reply:
I'll add a note to update the readme properly